libs/ui/src/components/rich-input.tsx

```diff
@@ -440,7 +444,16 @@ export const RichInput = forwardRef<Editor, RichInputProps>(
         Image,
         Underline,
         Highlight,
-        Link.configure({ openOnClick: false }),
+        Link.extend({
+          addKeyboardShortcuts() {
+            return {
+              "Mod-k": () => {
+                setLinkGlobal(this.editor as Editor);
+                return true;
```

Comment:
`addKeyboardShortcuts` expects functions in this object to return a `boolean`

![image](https://github.com/AmruthPillai/Reactive-Resume/assets/16601729/bb33fbb0-328a-44a0-b5e1-ba33c6808197)

![image](https://github.com/AmruthPillai/Reactive-Resume/assets/16601729/b1c6c6fe-a0fe-47d8-9fc8-672c691089a7)